### PR TITLE
Feature/webpack upgrades

### PR DESCRIPTION
**Pull request by Mimaaa**

### CODE REVIEW

File: app/routes/body.css [hunk 1]
```diff
@@ -1,10 +1,3 @@
-.body {
-  margin: 0;
-  padding: 0;
-  font-family: 'Avenir';
+.toedeledokie {
```
Comment (rijkvanzanten) [inline app/routes/body.css]: What's this, why is it here

File: records.json [hunk 2]
```diff
@@ -0,0 +1,237 @@
+{
```
Comment (rijkvanzanten) [inline records.json]: what's this file? what is it for? what does it do?

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
Nice changes. I'm wondering what all these extra files are for?

**Comment by rijkvanzanten** [comment]
Could you maybe add some documentation to this PR?

File: app/components/parts/genre/index.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import html from 'choo/html';
 import styles from '../parts.css';
 
 export const modalSettings = {
-  title: 'Hoi dit zijn instellingen tog',
+  title: 'Hoi dit zijn instellingen toch',
```
Comment (rijkvanzanten) [inline app/components/parts/genre/index.js:5]: 😂

File: package.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "nodemon --watch webpack.config.babel.js --exec \"webpack-dev-server --env.target development\"",
     "build": "webpack --env.target production",
-    "lint:js": "xo",
+    "lint:js": "eslint app/ webpack.*.js --cache",
```
Comment (rijkvanzanten) [inline package.json]: ❗️  This doesn't lint `server.js` and `api.js`

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (Mimaaa) [reply package.json]: Good one.

**Comment by Mimaaa** [comment]
records.json and stats.json are removed from the repo and are not being included from now on.

However, they serve a purpose:

**stats.json**
Importing the stats.json into a webpack analyser helps you understand the composition of your bundles and gives you more information on how to optimise the bundles.

**records.json**
"Records are particularly valuable if you have a complicated setup with code splitting and want to make sure the split parts gain correct caching behavior." SurviveJS - Webpack

The latter one is not that important for our setup, but it was nice to learn about it.

File: .eslintrc.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    sourceType: 'module',
+  },
+  rules: {
+    'comma-dangle': ['error', 'always-multiline'],
+    indent: ['error', 2],
+    'linebreak-style': ['error', 'unix'],
+    quotes: ['error', 'single'],
+    semi: ['error', 'always'],
+    'no-unused-vars': ['warn'],
+    'no-console': 0,
```
Comment (rijkvanzanten) [inline .eslintrc.js:19]: Maybe we should check for remaining console.log statements. We don't want to fill up the dev console with useless info when it isn't needed anymore

File: package.json [hunk 6]
```diff
@@ -36,8 +48,7 @@
     "webpack": "^2.6.1",
     "webpack-dev-server": "^2.4.5",
     "webpack-merge": "^4.1.0",
-    "xo": "^0.18.2",
-    "xo-loader": "^0.8.0"
+    "xo": "^0.18.2"
```
Comment (rijkvanzanten) [inline package.json]: Do we still need this dependency when switching to `eslint` completely?

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**